src/cli.cpp

```diff
@@ -58,6 +70,30 @@ void HelpAndQuit(cxxopts::Options options)
     exit(0);
 }
 
+string filename = "plot.dat";
+string tempdir = ".";
+string tempdir2 = ".";
+
+void cleanup(const string& tmpdir)
+{
+    string glob = fs::path(tmpdir) / (filename + ".*.tmp");
```

Comment:
I don't think it's safe to do all of these things in a signal handler.
See https://man7.org/linux/man-pages/man7/signal-safety.7.html
notably, [readdir](https://www.man7.org/linux/man-pages/man3/readdir.3.html) is not in that list.
It may also be tempting for future patches to add more logic here, that wouldn't be allowed. I think a much safer and robust way of handling signals is for the signal handler to just communicate back to the main thread(s) that it should terminate, and run the regular clean-up code there.

Reply:
Yes, cleanup should be done in DiskPlotter. I am thinking of ways to improve this so that is callable from Python (`chia plots create` process).

I am thinking the following methods would be good to have in DiskPlotter:

.stop()
.cleanup(phase = 1|2|..|all)

Additionally, would be great to have a way of pausing and resuming the plotting process or continuing partial plots, something like:

.pause()
.resume(plot_id, location) - resumes from last checkpoint (phase)

Reply:
Added `.stop()` and `.clean()` to DiskPlotter.

`.stop()` does not do anything currently but it should signal current phase to terminate (and I don't know how to do that atm)